scripts/make_api_rst.py

```diff
@@ -36,6 +36,55 @@
 Welcome to the QGIS Python API documentation project
 ==============================================================
 
+Introduction
+------------
+
+`QGIS <https://qgis.org>`_ is a user friendly Open Source Geographic Information System (GIS) that runs on Linux, Unix, Mac OSX, and Windows.
+QGIS supports vector, raster, and database formats. QGIS is licensed under the GNU General Public License.
+QGIS lets you browse and create map data on your computer. It supports many common spatial data formats (e.g. ESRI ShapeFile, GeoPackage, geotiff).
+QGIS supports plugins to do things like display tracks from your GPS. QGIS is Open Source software and its free of cost (download here).
+We welcome contributions from our user community in the form of code contributions, bug fixes, bug reports, contributed documentation,
+advocacy and supporting other users on our mailing lists and forums. Financial contributions are also welcome.
+
+QGIS source code is available at https://github.com/qgis/QGIS.
+There is also a `C++ version of the API documentation <https://api.qgis.org>`_ available.
+
+Earlier versions of the API
+---------------------------
+
+See `Backwards Incompatible Changes <https://api.qgis.org/api/master/api_break.html>`_ for information about incompatible changes to API between releases.
+
+Earlier versions of the documentation are also available on the QGIS website:
+`3.30 <https://qgis.org/pyqgis/3.30>`_,
+`3.28 (LTR) <https://qgis.org/pyqgis/3.28>`_,
+`3.26 <https://qgis.org/pyqgis/3.26>`_,
+`3.24 <https://qgis.org/pyqgis/3.24>`_,
+`3.22 (LTR) <https://qgis.org/pyqgis/3.22>`_,
+`3.20 <https://qgis.org/pyqgis/3.20>`_,
+`3.18 <https://qgis.org/pyqgis/3.18>`_,
+`3.16 (LTR) <https://qgis.org/pyqgis/3.16>`_,
+`3.14 <https://qgis.org/pyqgis/3.14>`_,
+`3.12 <https://qgis.org/pyqgis/3.12>`_,
+`3.10 (LTR) <https://qgis.org/pyqgis/3.10>`_,
+`3.8 <https://qgis.org/pyqgis/3.8>`_,
+`3.6 <https://qgis.org/pyqgis/3.6>`_,
+`3.4(LTR) <https://qgis.org/pyqgis/3.4>`_,
+`3.2 <https://qgis.org/pyqgis/3.2>`_,
+`3.0 <https://qgis.org/pyqgis/3.0>`_,
+
+Communication channels
+----------------------
+
+For support we encourage you to join our `mailing lists <https://qgis.org/en/site/forusers/support.html#mailing-lists>`_ for users and developers.
+Some QGIS users and developers can also often be found in channels such as Matrix, Telegram,...
+
+Bug Reporting
+--------------
+
+If you think you have found a bug, please report it using our `bug tracker <https://github.com/qgis/pyqgis/issues>`_.
```

Comment:
Worth mentioning "a bug in documentation" ? 
I think current formulation would lead people to report QGIS issue in pyqgis repo.

Reply:
I must confess I don't really know how when the report should g to QGIS and when to pyQGIS, and I feel like there are not many people watching this repo (despite it is of high importance)
```suggestion
If you think you have found a bug in the documentation, please report it using our `bug tracker <https://github.com/qgis/pyqgis/issues>`_.
```